nixos/modules/services/continuous-integration/buildbot/worker.nix

```diff
@@ -115,7 +116,11 @@ in {
         User = cfg.user;
         Group = cfg.group;
         WorkingDirectory = cfg.home;
-        ExecStart = "${cfg.package}/bin/buildbot-worker start --nodaemon ${cfg.buildbotDir}";
+        Environment = "PYTHONPATH=${cfg.package}/lib/python2.7/site-packages:${pkgs.python27Packages.future}/lib/python2.7/site-packages";
```

Comment:
@FRidh does this make sense regarding calling `twistd` natively (as suggested above in comments... previously calling `buildbot start --nodaemon`)? is there another way to set PYTHONPATH other than the systemd's unit Environment? perhaps an example of `provide a Python on PATH that has everything set up already`?